fs/Kconfig

```diff
@@ -111,6 +111,13 @@ config SENDFILE_BUFSIZE
 	---help---
 		Size of the I/O buffer to allocate in sendfile().  Default: 512b
 
+config FS_HEAPSIZE
+	int "Independent heap bytes used by shm/tmpfs/pseudofile"
+	default 0
```

Comment:
how about protected/kernel mode?

Reply:
It's optional and default 0 disabled, if not prefer use independ fs heap, It is just ignore and will pass malloc to kmm directly. same behaviour as before commit.

and as the fs_heap.c only used kmm & mm_initialize API, should also compatible with protected/kernel mode, if prefer open it.

even in protected/kernel mode, shm/tmpfs/pseudofile will also be global, no need multi-instance.